src/components/Chat/AvailableVolunteersComponent.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+interface IProps {
+  availableVolunteers: string[];
+  closePopup(): any;
+}
+
+const AvailableVolunteersComponent = (props: IProps) => {
+  const selectUser = name => {
+    console.log(name);
+  };
+
+  const mapAvailable = () =>
```

Comment:
Kan være en ide å bruke useMemo hooken her for å memoizere, men ikke nødvendig. isåfall vil availableVolounteers bli brukt som dependency. typ:

`const mapAvailable = useMemo(() => availableVolounteers.map(<det som skjer i koden>), [availableVoulounteers])`

På denne måten blir det ikke rerendret uten at det trengs